src/Redux/rockets/reducer.js

```diff
@@ -0,0 +1,13 @@
+import { showRockets } from './action-makers';
+import { rocketShowed } from './action-types';
+
+const showRocket = function reducer(state = [], action) {
+  switch (action.type) {
+    case (rocketShowed):
+      return action.payload;
```

Comment:
Would suggest spreading your data return [... action.payload] , to add your entire array of data.

Reply:
this will return an array also you can check the result at the store after dispatching